libs/ui/src/lib/tabs/tabs.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="cvi-tabs__list" role="tablist">
+  <button
+    *ngFor="let tab of allTabs; let i = index"
+    dataAttribute="tab-button-{{ i }}"
+    [ngClass]="{ active: isTabSelected(i), underlined: isTabSelected(i) }"
+    (click)="makeActive(i)"
+    type="button"
+    #tabButton
+    id="{{ baseId }}-tab-{{ i }}"
+    [attr.aria-selected]="isTabSelected(i)"
+    role="tab"
+    [attr.tabindex]="isTabSelected(i) ? null : '-1'"
+    [attr.aria-controls]="baseId + '-tabpanel-' + i"
+    class="cvi-tabs__tab-btn"
+  >
+    {{ tab.title }}
+  </button>
+</div>
+<div class="cvi-tabs__list-mobile" role="menu">
+  <button
+    role="button"
+    class="cvi-tabs__mobile-button"
+    type="button"
+    id="{{ baseId }}-tab-button}"
+    aria-expanded="false"
+    (click)="handleArrowButtonClick($event)"
+  >
+    {{ getActiveTabTitle() }}
+    <cvi-ng-icon
+      [class]="'cvi-tabs__icon'"
+      [name]="isOpen ? 'arrow_up_alt' : 'arrow_down_alt'"
+      [svgClass]="'cvi-tabs__arrow-icon'"
+    >
+    </cvi-ng-icon>
+  </button>
+</div>
+<div
+  *ngIf="isOpen"
+  dataAttribute="tabs-popup"
+  class="cvi-tabs__dropdown-popup"
+  #popup
+>
+  <div class="cvi-tabs__dropdown">
+    <ul
+      role="listbox"
+      class="cvi-tabs__dropdown-container"
+      (closed)="close()"
+      role="menu"
+      (keydown)="handleKeyboardEvent($event)"
+    >
+      <button
+        *ngFor="let tab of allTabs; let i = index"
+        class="cvi-tabs__dropdown-item"
+        dataAttribute="tab-button-{{ i }}"
+        (click)="makeActive(i)"
+        id="{{ baseId }}-tab-{{ i }}"
+        [attr.aria-selected]="isTabSelected(i)"
+        role="menuitem"
+        [attr.aria-controls]="baseId + '-menuitem-' + i"
+        #menuButton
+      >
+        {{ tab.title }}
+      </button>
+    </ul>
+  </div>
+</div>
+<div
+  dataAttribute="tab-content"
+  class="cvi-tabs__tab-content"
+  [attr.aria-labelledby]="baseId + '-tab-' + activeIndex"
```

Comment:
We need to add both tab button and dropdown button ids to this attribute, space-separated:
```suggestion
  [attr.aria-labelledby]="baseId + '-tab-' + activeIndex + ' ' + baseId + '-dropdown-button-' + activeIndex"
```